### PR TITLE
WX-844-845 Rename WorkflowDashboard to RunDetails, created header sub-component

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [WX-844](https://broadworkbench.atlassian.net/browse/WX-844) and [WX-845](https://broadworkbench.atlassian.net/browse/WX-845)

PR renames `WorkflowDashboard` to `RunDetails` and updates all associated language regarding those terms. PR also creates a header component for both `SubmissionDetails` and `RunDetails` and updates styling and language on `RunDetails` to match that shown in the mock.

Also there's some additional import cleanups to switch out entire module imports for selective ones as well as an update to `react-hyperscript-helpers`

### CODE REVIEW

File: src/pages/workspaces/workspace/jobHistory/CallTable.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const CallTable = ({ namespace, name, submissionId, workflowId, callName, callOb
           {
             size: { basis: 200, grow: 2 },
             field: 'callCachingResult',
-            headerRenderer: () => h(Sortable, { sort, field: 'callCachingResult', onSort: setSort }, ['Call Caching Result']),
+            headerRenderer: () => h(Sortable, { sort, field: 'callCachingResult', onSort: setSort }, ['Task Caching Result']),
```
Comment (jgainerdewar) [inline src/pages/workspaces/workspace/jobHistory/CallTable.js:97]: "Call caching" isn't the most intuitive name but it's something users are familiar with, we should leave this as-is.

File: src/pages/SubmissionDetails.js [hunk 2]
```diff
@@ -39,26 +40,32 @@ export const SubmissionDetails = ({ submissionId }) => {
...
-        h1(['Submission Details']),
-        h(ButtonOutline, {
-          onClick: () => Nav.goToPath('root')
-        }, ['Submit a new workflow'])
-      ]),
-      div([
-        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`])
-      ]),
+  return div({ id: 'submission-details-page' }, [
+    Navbar('SUBMIT WORKFLOWS WITH CROMWELL'),
```
Comment (jgainerdewar) [inline src/pages/SubmissionDetails.js:66]: I'd give it even odds that the inconsistent headers in the mocks are an error, or that we'll decide to make it consistent later. For now, though, flexibility FTW.

File: src/libs/routes.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
-import _ from 'lodash/fp'
+import { flatten, map } from 'lodash/fp'
 import { compile, pathToRegexp } from 'path-to-regexp'
 import { routeHandlersStore } from 'src/libs/state'
 import * as SubmissionConfig from 'src/pages/SubmissionConfig'
 import * as SubmissionDetails from 'src/pages/SubmissionDetails'
 import * as SubmissionHistory from 'src/pages/SubmissionHistory'
 import * as SubmitWorkflow from 'src/pages/SubmitWorkflow'
-import * as WorkspaceDashboard from 'src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js'
+import * as RunDetails from 'src/pages/workspaces/workspace/jobHistory/RunDetails.js'
```
[the remark this answers is not in the document]
Comment (JVThomas) [reply src/libs/routes.js:8]: Ah, if it's okay to move it then yeah, I'd rather have it under `src/pages` since it's a page component

**Review by jgainerdewar — APPROVED** [review]
Looks like there are some merge conflicts, but once those are resolved I think this is good to go!

**Review by kraefrei — APPROVED** [review]

**Pull request merged**